src/components/Profile/ProfileUserDetails/ProfileUserDetails.js

```diff
@@ -69,16 +81,24 @@ export class ProfileUserDetails extends Component {
               <span className="following">
                 <span className="number">{120}</span> Following
               </span>
+              <div className="divider" />
+              <span className="block medium-v-padding">
+                Reading stats:{' '}
+                <Button buttonClass="button radius-2 small-padding yellow">
```

Comment:
@luc-tuyishime, as I see this button only displays the number of articles read. If it could be possible to also display a list of read articles

Reply:
The listing of articles is not required but I will implement it once I am done with all my feature. Thank You.